### PR TITLE
Enhance schema style setting

### PR DESCRIPTION
**Pull request by Linary**

### CODE REVIEW

File: hubble-be/src/main/java/com/baidu/hugegraph/service/schema/SchemaService.java [hunk 1]
```diff
@@ -58,13 +58,7 @@
...
 
     public static final String USER_KEY_CREATE_TIME = "create_time";
-    public static final String USER_KEY_ICON = "~icon";
-    public static final String USER_KEY_COLOR = "~color";
-    public static final String USER_KEY_SIZE = "~size";
-    public static final String USER_KEY_ARROW = "~with_arrow";
-    public static final String USER_KEY_THICK = "~thickness";
-    public static final String USER_KEY_FIELDS = "~display_fields";
-    public static final String USER_KEY_SYMBOLS = "~join_symbols";
+    public static final String USER_KEY_STYLE = "~style";
```
Comment (javeme) [inline hubble-be/src/main/java/com/baidu/hugegraph/service/schema/SchemaService.java]: why the 'create_time' isn't prefixed with '~'

File: hubble-be/src/main/java/com/baidu/hugegraph/util/HugeClientUtil.java [hunk 2]
```diff
@@ -22,22 +22,26 @@
...
 
-    public static HugeClient tryConnect(GraphConnection connection) {
+    public static HugeClient tryConnect(GraphConnection connection,
+                                        HugeConfig config) {
         String graph = connection.getGraph();
         String host = connection.getHost();
         Integer port = connection.getPort();
         String username = connection.getUsername();
         String password = connection.getPassword();
+        int timeout = config.get(HubbleOptions.CLIENT_REQUEST_TIMEOUT);
```
Comment (javeme) [inline hubble-be/src/main/java/com/baidu/hugegraph/util/HugeClientUtil.java]: add timeout to GraphConnection

File: hubble-be/src/main/resources/i18n/messages_zh_CN.properties [hunk 3]
```diff
@@ -44,6 +44,7 @@ gremlin.execute.failed=Gremlin 执行失败，详细信息: {0}
 gremlin.execute.timeout=Gremlin 执行超时，详细信息: {0}
 gremlin.connection.refused=无法连接到 HugeGraphServer, 请检查服务是否可用
 gremlin.convert-vertex-id.failed=转换顶点 id {0} 失败（按照 id 策略 {1}）
+gremlin.edges.linked-vertex.not-exist=边集合中有边的源顶点或目标顶点不存在
```
Comment (javeme) [inline hubble-be/src/main/resources/i18n/messages_zh_CN.properties]: 部分边的源顶点或目标顶点不存在

File: hubble-be/src/main/java/com/baidu/hugegraph/service/schema/VertexLabelService.java [hunk 4]
```diff
@@ -385,13 +398,17 @@ private static VertexLabel convert(LabelUpdateEntity entity,
...
-        }
-        if (style.getColor() != null) {
-            builder.userdata(USER_KEY_COLOR, style.getColor());
+
+        VertexLabel vertexLabel = builder.build();
+        Map<String, Object> userdata = vertexLabel.userdata();
+
+        // The style requires the front end to pass in the full amount
+        // TODO: use builder or setter, Now use builder throw
+        //  Can't access builder which is completed
```
Comment (zhoney) [inline hubble-be/src/main/java/com/baidu/hugegraph/service/schema/VertexLabelService.java]: Now -> now
“throw”？
delete empty before “Can't”

File: hubble-be/src/main/resources/i18n/messages.properties [hunk 5]
```diff
@@ -44,6 +44,7 @@ gremlin.execute.failed=Gremlin execute failed, the details: {0}
 gremlin.execute.timeout=Gremlin execute timeout, the details: {0}
 gremlin.connection.refused=Can't connect to HugeGraphServer, please ensure it's available
 gremlin.convert-vertex-id.failed=Failed to convert vertex id {0} according to its id idstrategy {1}
+gremlin.edges.linked-vertex.not-exist=The edges' source/target vertices doesn't exist
```
Comment (zhoney) [inline hubble-be/src/main/resources/i18n/messages.properties]: vertices -> vertex

**Pull request merged**